Magento2/Tests/Legacy/ObsoleteAclUnitTest.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+-->
+<config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:framework:Acl/etc/acl.xsd">
+    <children>
+        <title value="foo"/>
+    </children>
+    <title value="bar"/>
+    <children default_policy="deny"/>
+    <acl>
+        <children>
+            <title value="foo"/>
```

Comment:
Copy this file into a new one and put the attribute `value` in a new line, to check that it will get errors in weird formatted XML files

Reply:
I have added a new line inside the same file following your recommendation. I tested it and it works correctly as it matches the pattern provided.